### DISCUSSION TITLE
Improve information to join EddieHubCommunity organisation.

### DISCUSSION THREAD

**Discussion by DevCassie** [EddieHubCommunity/support#1064] [Ideas]
There has been an idea to improve the information on where to join this organisation, as it resulted in a few confusions.

The following suggestions have been made already:
* @kaiwalyakoparkar suggested pinning the support repository to the top. 
* @eddiejaoude suggested adding a line to every repository's readme that will tell you how to join the organisation. 
* @DevCassie suggested adding a line in the organisation's description. 

Perhaps there are some other ideas. What are your insights?

**Comment by eddiejaoude** [comment]
the MEE6 welcome message is currently

> Have a great time here in **{server}**. We are a friendly and inclusive community. Please set a profile picture (it can be a pet or cartoon character if you prefer not to have your face on there), dont forget to play nicely and collaborate with the community. If you have any issues please let one of the Moderators aware, if you are still having issues or can not reach a moderate, please contact @moderators  . Lets geek out!

**Reply by kaiwalyakoparkar** [reply]
Ok, so we can add something like "If you are not a member of eddiehub on github. What are you waiting for join it here <link>" . Will that work?

**Comment by Frakko** [comment]
Going over the points that @DevCassie and @eddiejaoude listed above:

- I personally don't see pinning the repository of much help, as it already shows up between the top 3/4 repositories and it's (usually) easy to see - also because its description presents it as a 'support' repository thus people that are looking for help are very likely to open it.
- I find adding a message in **every** repository excessive - I suggest adding some informatioons in the main 'community-oriented' repositories (like awesome-github-profiles and hacktoberfest-practice).
- Adding informations about it in the description of the organisation would help, however I do like the clean one-line look that it has now. 😅
- I do like the idea of adding some informations to MEE6's join message, as people new to GitHub and unconfortable with the enviroment, could join the Discord server before looking at the organisation, and it would probably better catch their attention, as it's a private message, rather than a few lines in the readme channel in the Discord server that many people overlook.

Finally, I suggest asking the people that are/were encountering issues joining the community, how they got to know about it. For example, if many people found out about it through one of Eddie's YouTube videos' description and don't exactly know how to join, it's *likely* that that's where the problem resided, thus adding some instructions there could help.

These are just my personal opinions - feel free to give me your thoughts one it! 🙂

**Reply by DevCassie** [reply]
That sounds like an idea 👍 . I must agree that the one-lined description looks good.

If for some reason we decide to add it to the join message of MEE6, then what about the people that won't join the discord server or won't be able to join the discord? What would be a solution?

**Reply by starlightknown** [reply]
how about putting that up in the org's description? That will be directly visible and we don't have to add it to every repository. What do you think?

**Reply by starlightknown** [reply]
wait that idea has been ruled out, I didn't see 
okay so we can add that in Youtube video description as said above

**Reply by eddiejaoude** [reply]
I like the idea of adding it to the Org description, but I appreciate we don't want this too long. I tried this, what do you think?

<img width="741" alt="Screenshot 2021-04-06 at 07 41 32" src="https://user-images.githubusercontent.com/624760/113669398-9ea32c80-96ab-11eb-88c7-7fc6fe1f86ca.png">

**Reply by starlightknown** [reply]
It was better before

**Reply by eddiejaoude** [reply]
I can't remember what it was before 🤦‍♂️

**Reply by starlightknown** [reply]
Before you tried adding `community of inclusive opensource people`. It was just `collaboration and code second` this is before. 

> I like the idea of adding it to the Org description, but I appreciate we don't want this too long. I tried this, what do you think?
> 
> <img alt="Screenshot 2021-04-06 at 07 41 32" width="741" src="https://user-images.githubusercontent.com/624760/113669398-9ea32c80-96ab-11eb-88c7-7fc6fe1f86ca.png">

Refer this

**Comment by eddiejaoude** [comment]
Could an issue be made with some action points? I think I have to make some of the admin changes
